incubator/hnc/e2e/tests/e2e_test.go

```diff
@@ -39,7 +39,44 @@ func tryRun(cmdln ...string) error {
 	return err
 }
 
-func runShouldNotContain(substr string, timeDuration string, cmdln ...string) {
+func runShouldContain(substr string, seconds float64, cmdln ...string) {
```

Comment:
No copy-paste :D There's a much shorter way to implement `runShouldContain` now that `runShouldContainMultiple` exists!

Reply:
ah good catch. Done